readme.mkd

```diff
@@ -190,6 +191,37 @@ If you need secure communication, e.g. are syncing across the open Internet with
 
 This will have the mirror client send traffic to the `localhost:49172` port, which SSH will securely tunnel to your remote host.
 
+Running with Docker
+===================
+
+A docker image is available at [quay.io/stephenh/mirror](https://quay.io/stephenh/mirror).
+
+Since the container will have its own filesystem separate from the host's filesystem,
+usually you'll want to mount some directory *into* the container to make it available for
+synchronisation.
+To mount the current working directory as `/data` into the container, pass `-v $(pwd):/data` to docker.
+
+The image accepts two environment variables to control the owner of the files that *mirror* will create:
```

Comment:
Is the owner of the files as-exposed-to-the-container, or the owner as-exposed-to-our-real-filesystem? I imagine it is the latter? Can you clarify that it will be the owner of the files on the external/non-docker file system if so?

Reply:
Also, can you describe what happens if U/G are not set, e.g. will it default to the owner being user/group of whoever started the docker container? That would be my guess but I'm not sure.